solr/README.md

```diff
@@ -5,27 +5,66 @@ All indexing is done on the "reindexing solr", currently
 every night to the production servers, which we never actually need to 
 touch.
 
-Deploying a new solr configuration is as follows:
+Deploying a new solr configuration is as follows. First, get the new code:
 
 ```shell
 cd /htsolr/catalog/bin/ht_catalog_indexer
-git pull origin main # get the new code
+# verify there is nothing uncommitted with git status; commit or git stash as needed
```

Comment:
"It's expected that the translation maps (e.g., `collection_code_to_original_from.yaml`) will have changed, as they're generated from the database on each run."

Reply:
Added a note.